.mailmap

```diff
@@ -588,6 +588,8 @@ Gilbert Gede <gilbertgede@gmail.com> <ggede@ucdavis.edu>
 Gilles Schintgen <gschintgen@hambier.lu>
 Gina <Dr-G@users.noreply.github.com>
 Gleb Siroki <g.shiroki@gmail.com>
+Glenn Horton-Smith <glenn.hortonsmith@gmail.com>
+Glenn Horton-Smith <glenn.hortonsmith@gmail.com> Glenn Horton-Smith <gahs@phys.ksu.edu>
```

Comment:
I think only the first of these lines is needed. It doesn't look like the `ksu.edu` email is used in any of the commits. Did you add that because of an error reported by the mailmap script?

If you actually wanted your entry in the AUTHORS file to be recorded under `ksu.edu` then it should be the other way around i.e.:
```
Glenn Horton-Smith <gahs@phys.ksu.edu> Glenn Horton-Smith <glenn.hortonsmith@gmail.com>
```
That way it is saying that the gmail address is an alias for the ksu address which should be the one recorded in the AUTHORS file and associated with these commits.

Reply:
It's not needed for any current commit. I can remove the second line if you like. I only did it because I have "email = gahs@phys.ksu.edu" as a global setting on my work computer and I thought it might be useful in case I ever made a commit from work without remembering to set the email correctly.

Reply:
The question is just which email address you would like to be recorded in AUTHORS. If you want it to be the gmail address then the above is fine.

Reply:
Ah, ok. Yes, I want it to be the gmail address. Thanks for asking.